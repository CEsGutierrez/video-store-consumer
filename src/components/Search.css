.searchList {
  display: flex;
  flex-wrap: wrap;
  justify-content: center;
}

.search {
  background-color: rgba(181, 181, 181, 50%);

  padding: 1em 0;
  margin: 0.5rem;

  min-height: 20%;
  min-width: 50%;
  flex-basis: 18%;

  border-radius: 5px;

  display: grid;
  grid-template-columns: 1fr 5fr 1fr;
  align-items: center;
}

.search-card {
  background-color:  rgba(181, 181, 181, 50%);
  padding: 1em;
  margin-top: 1em;
}

.search__content {
  grid-column-start: 2;

  display: flex;
  flex-direction: column;
  justify-content: space-around;

  text-align: center;
  font-weight: 100;
  overflow: hidden;
  font-family: 'Courier New', Courier, monospace;
}

.search__content-bar {
  width: 100%;
  height: 30px;
  border-radius: 20px;
  border: none;
}

.search__content-bar::placeholder {
  text-align: center;
  font-family: 'Courier New', Courier, monospace;
}

.search__content-bar, .search__content-button {
  padding: 0;
  margin: 0;
}

.search__content-button {
  margin-top: 15px;
  align-self: center;
  font-family: 'Courier New', Courier, monospace;
  font-size: larger;
  width: 100px;
  height: 50px;
  border-radius: 50px;
  background: rgb(181, 181, 181, 50%);
  border: none;

}

.search__content-button:hover {
  cursor: pointer;
}

.add-movie-button {
  margin-top: 15px;
  align-self: center;
  font-family: 'Courier New', Courier, monospace;
  background-color: rgb(181, 181, 181);
  border: none;
  border-radius: 10px;
  font-weight: bold;
}
