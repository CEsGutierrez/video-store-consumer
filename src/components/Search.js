import React from 'react';
// import PropTypes from 'prop-types';

const Search = () => {
  return <h2>Search</h2>;
}

export default Search