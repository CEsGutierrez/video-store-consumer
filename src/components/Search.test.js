import React from 'react';
import { render, cleanup } from '@testing-library/react'
import Search from './Search';

describe('Search', () => {
  test('that it matches the existing snapshot', () => {
    // Arrange-Act
    const { asFragment } = render(
      <Search
      searchExternalCallback={() => { }}
      searchResultsCallback={() => { }}
      addMovieCallback={() => { }}
      resetMessageCallback={() => { }}
      />
    );

    // Assert
    expect(asFragment()).toMatchSnapshot();
    cleanup();
  });
});