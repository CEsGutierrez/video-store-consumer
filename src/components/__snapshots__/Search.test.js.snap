// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search that it matches the existing snapshot 1`] = `
<DocumentFragment>
  <div>
    <form
      class="new-pet-form"
    >
      <div>
        <label
          class=""
          for="search"
        >
          Search: 
        </label>
        <input
          id="search"
          name="searchTerm"
          placeholder="Enter a movie title"
          value=""
        />
      </div>
      <input
        class=""
        name="submit"
        type="submit"
        value="Submit"
      />
    </form>
    <section>
        
    </section>
  </div>
</DocumentFragment>
`;
